### DISCUSSION TITLE
Aggressive monitor wake querying

### DISCUSSION THREAD

**Discussion by Aidandsteele** [MonitorControl/MonitorControl#1264] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

This is not a problem with the app as it exists now.
I have this Philips [monitor](https://www.amazon.com/gp/product/B08KFSMGJ8/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1)  and it is super slow or fails entirely to wake from sleep when my macbook is in clamhell mode. I know the computer wakes up with mouse movement because I get an alert that my apple watch has unlocked it. Apparently other people have had similar [issues](https://discussions.apple.com/thread/253716798)

### Describe the solution you'd like

It would be amazing if we could work together to figure out what is failing to happen that is causing my monitor not to wake up. I hope to help you develop a feature that queries the monitor in a way that makes it wake from sleep.

### Describe alternatives you've considered

I've tried various solutions from other forums; high-speed HDMI cable, safe boot, etc.

### Anything else?

I'm not a regular github user so if you want to collaborate please contact me at steelead@gmail.com.

**Comment by waydabber** [comment]
I am not sure if this is something MonitorControl can fix as this seems to be an OS/firmware/hardware level issue, but many displays do things with the Mac, that's true (I am now using a CoolerMaster GPU27U which does the same...).
